python/lsst/ip/isr/isrTask.py

```diff
@@ -375,19 +380,36 @@ def readIsrData(self, dataRef, rawExposure):
         else:
             fringeStruct = pipeBase.Struct(fringes=None)
 
+        if self.config.doAttachTransmissionCurve:
+            opticsTransmission = dataRef.get("transmission_optics")
+            filterTransmission = dataRef.get("transmission_filter")
+            sensorTransmission = dataRef.get("transmission_sensor")
+            atmosphereTransmission = dataRef.get("transmission_atmosphere")
```

Comment:
Each of the above is supposed to be optional. Do you want to have toggles for each of them separately? Maybe `try` to read and set them to `None` if they don't exist?

Reply:
I like the `try` idea (too bad you cant' pass a default to `butler.get` like you can with `dict.get`).

Reply:
Hmm, actually, we should probably make it a bunch of config options so it's a hard failure rather than a silently incorrect curve when dataset is missing.